### PR TITLE
Add Polish translation

### PR DESCRIPTION
**Pull request by adrianjagielak**
Using original macOS translations and some own

### CODE REVIEW

File: HeliPort.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -117,6 +117,7 @@
 		F8F6CF0B243D675800965B43 /* Api.c */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.c; path = Api.c; sourceTree = "<group>"; };
 		F8F6CF0E243D67D800965B43 /* IoctlId.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = IoctlId.h; sourceTree = "<group>"; };
 		FD651AE794A9A932898331AD /* Pods-HeliPort.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-HeliPort.debug.xcconfig"; path = "Target Support Files/Pods-HeliPort/Pods-HeliPort.debug.xcconfig"; sourceTree = "<group>"; };
+		FE29EB4F24B9CE1200285757 /* pl */ = {isa = PBXFileReference; lastKnownFileType = text.plist.strings; name = pl; path = pl.lproj/Localizable.strings; sourceTree = "<group>"; };
```
Comment (williambj1) [inline HeliPort.xcodeproj/project.pbxproj:120]: Please specify file encoding and line ending

image

Comment (adrianjagielak) [reply HeliPort.xcodeproj/project.pbxproj]: Done

**Comment by adrianjagielak** [comment]
Specified file encoding and line ending

**Comment by williambj1** [comment]
Looks good, merged.

**Pull request merged**